### PR TITLE
[MKC-1158] MKR zero tutorial import

### PR DESCRIPTION
**Pull request by BenjaminDannegard**
## What This PR Changes
- Added lost tutorial to MKR Zero

## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by karlsoderby — APPROVED** [review]
LGTM!

File: content/hardware/01.mkr/01.boards/mkr-zero/tutorials/mkr-zero-garden-automation/content.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
+---
+title: 'Home Gardening Automation with MKR Zero'
+difficulty: intermediate
+compatible-products: [mkr-zero]
+description: "Do you like home gardening but you've never had a green thumb? This could be the right solution for you!"
+tags:
+  - Automation
+  - Gardening
+author: 'Benjamin Dannegård'
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/01.boards/mkr-zero/tutorials/mkr-zero-garden-automation/content.md:9]: @BenjaminDannegard please update the name following our conversation :)

File: content/hardware/01.mkr/01.boards/mkr-zero/tutorials/mkr-zero-garden-automation/content.md [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+- Seeed Seed - Grove - Oled Display 0.96"
+- Seeed Seed - Grove - Water Sensor
+- Seeed GROVE - HIGH PRECISION BAROMETRIC PRESSURE SENSOR (DPS310)
+- Seeed Grove - I2C Hub
+- Arduino MKR Connector Carrier (Grove compatible)
+- Arduino USB cable type A male to micro type B male
+
+### Circuit
+
+This project is based on the Arduino MKRZERO, and few other Grove components ready-to-use; the BOM is the following:
```
Comment (jhansson-ard) [inline content/hardware/01.mkr/01.boards/mkr-zero/tutorials/mkr-zero-garden-automation/content.md:37]:
```suggestion
This project is based on the Arduino MKR Zero, and few other Grove components ready-to-use; the BOM is the following:
```

**Comment by jhansson-ard** [comment]
@BenjaminDannegard please update the name so that we can then merge this PR 🙏🏼

**Pull request merged**